website/docs/externaldata.md

```diff
@@ -146,6 +146,9 @@ If there is a system error, the provider should return the system error message
 
 Example provider implementation: https://github.com/open-policy-agent/gatekeeper/blob/master/test/externaldata/dummy-provider/provider.go
 
+#### Caching
+Gatekeeper now supports caching of responses from external data providers. It caches the response based on the `Key` and `Value` received as part of the [`ProviderResponse`](#providerresponse). By default, the cache is invalidated after 3 minutes, which is the default Time-to-Live (TTL). You can configure the TTL using the --external-data-provider-response-cache-ttl flag.
```

Comment:
```suggestion
Starting with v3.13+, Gatekeeper supports caching of responses from external data providers. It caches the response based on the `Key` and `Value` received as part of the [`ProviderResponse`](#providerresponse). By default, the cache is invalidated after 3 minutes, which is the default Time-to-Live (TTL). You can configure the TTL using the `--external-data-provider-response-cache-ttl` flag.
```

Reply:
Do we want to give the version when a feature arrives? Is that implicit in the versioning of docs?

Definitely prefer listing a specific version to "now supports"

Reply:
+1 on adding something like

> `Feature State`: Gatekeeper version v3.11+ (beta)

Reply:
SGTM

Reply:
I added `Starting with v3.13+, Gatekeeper supports...` per @sozercan suggestion. Does that work?